targets/arduino-mega2560.json

```
@@ -0,0 +1,8 @@
+{
+    "inherits": ["atmega2560"],
+    "build-tags": ["arduino_mega2560"],
+    "ldflags": [
+        "-Wl,--defsym=_bootloader_size=253952"
```

It's the other way round: 253952 is the size available to the program (according to boards.txt in the Arduino IDE). The chip has 256kB of memory (262144) so the bootloader size must be 8192, or 8K.

```suggestion
        "-Wl,--defsym=_bootloader_size=8192"
```

---

Ok I made this change.  I guess I misunderstood an earlier conversation when I put that in place.

---

Where do I place the `253952` instead?  Is there another `defsym` value?

---

Nowhere, it's just a different way of indicating which area of flash is available for the program.